common/src/main/scala/uk/ac/wellcome/platform/archive/common/storage/services/memory/MemoryStorageManifestDao.scala

```diff
@@ -0,0 +1,16 @@
+package uk.ac.wellcome.platform.archive.common.storage.services.memory
```

Comment:
Do you think these memory versions should be in the test namespace? I think it makes sense for them to **not be** in the library, but here it feels like they should never be used except in tests.

Reply:
I have no strong feelings, but it feels a bit odd to be inconsistent across the two libraries. Either memory implementations are for testing or they’re not.